### PR TITLE
Added guidance on omitting part of an output

### PR DESCRIPTION
**Pull request by julian-cable**

### CODE REVIEW

**Review by daobrien — CHANGES_REQUESTED** [review]
I think you need to remember that this is a public style guide used by people outside of Red Hat, and in these situation we should offer advice, not presciptions. Consequently, I'd use language such as "You can use ...output omitted... or similar to indicate content that has been omitted." OR, state what RH uses but add that "you can use whatever suits your organization, but be as consistent as possible blah blah".

In our adoc guide we definitely need to state exactly what, how, when, etc., to use, because that guide is only for RHT.

**Comment by julian-cable** [comment]
@daobrien I've updated the wording here. Please re-review.

File: en-US/Design.xml [hunk 1]
```diff
@@ -484,7 +484,8 @@ $ vi myFile.txt
           <para>
             For the sake of brevity, do not show all output to the user in all cases, but only those parts of any output that are relevant to the context that is described. 
 			Where output is not included, place a marker to show that information is purposely excluded.
-			When shortening the output, use the <emphasis>...output omitted...</emphasis> notation, starting and ending with an ellipsis, and highlight it in italics.
+			When shortening the output, use a consistent notation.
+			Red{nbsp} uses the <emphasis>...output omitted...</emphasis> notation, starting and ending with an ellipsis, and highlighted in italics.
```
Comment (daobrien) [inline en-US/Design.xml:488]:
```suggestion
			Red{nbsp}Hat uses the <emphasis>...output omitted...</emphasis> notation, starting and ending with an ellipsis, and highlighted in italics.
```

**Review by daobrien — APPROVED** [review]
Approving, assuming ur going to call us "Red Hat" in the final version ;-)

**Comment by julian-cable** [comment]
@daobrien Oops, now fixed!

**Pull request merged**